addons/purchase_stock/tests/test_replenish_wizard.py

```diff
@@ -492,3 +493,28 @@ def test_correct_supplier(self):
             ('partner_id', '=', partner_b.id)
         ])
         self.assertEqual(po.amount_untaxed, 10, "best price is 10$")
+
+    def test_inter_wh_replenish(self):
```

Comment:
Add a brief description on what the test is supposed to check.
In this case, `inter_wh_replenish` is a bit vague :sweat_smile: 

```suggestion
    def test_inter_wh_replenish(self):
        """ e.g. This tests makes sure that something happens correctly.
        """
```